Dockerfile

```diff
@@ -4,4 +4,6 @@ USER root
 RUN apt-get -y update && apt-get -y install git
 USER airflow
 
-COPY --chown=airflow:root ./dags /opt/airflow/dags
\ No newline at end of file
+COPY --chown=airflow:root dist/ils_middleware-0.1.0-py3-none-any.whl .
```

Comment:
I think there should be a better method to install `ils_middleware` in the image. I need to do some more investigation.